.github/test_sdk/main.cxx

```diff
@@ -1,10 +1,16 @@
 #include <f3d/engine.h>
+#include <f3d/image.h>
+#include <f3d/interactor.h>
+#include <f3d/loader.h>
+#include <f3d/log.h>
+#include <f3d/options.h>
+#include <f3d/window.h>
 
 int main()
 {
-  f3d::engine eng(f3d::engine::CREATE_WINDOW | f3d::engine::CREATE_INTERACTOR);
+  f3d::engine eng(f3d::engine::FLAGS_NONE);
 
-  // TODO Actual test
+  f3d::log::info("F3D engine is loaded")
```

Comment:
what about more than just creating an empty engine ? Loading a file and rendering it would be better and will check that the installation contains more than just headers and a few engine symbols.

Reply:
I still think current test is a bit lacking. What is wrong by adding a simple load and render test instead ?

Reply:
In practice, we don't have a file available to open here, we do not know the path to the source dir. I can create an obj file manually but it seems complicated for something that is tested in other tests.

Reply:
Indeed, I did not consider that. That is fine then.